site/en/tutorials/load_data/images.ipynb

```diff
@@ -1,4 +1,4 @@
-{
+v{
```

Comment:
Looks like this parse error sneaked in

Reply:
Sorry for that. I also doubt that this explanation about `repeat` and `shuffle` has a conflict with this [document](https://www.tensorflow.org/guide/performance/datasets).
The detail is seen in the [stackoverflow question](https://stackoverflow.com/questions/55851440/is-there-a-conflict-in-these-two-tensorflow-documents-about-repeat-and-shuffle) I asked.